installer/installer.py

```diff
@@ -96,6 +103,10 @@ def search_and_replace(search_text, replace_text, file_path):
             
     file.close()
 
+def version_tuple(v):
+    return tuple(map(int, (v.split("."))))
```

Comment:
Just strip `v` if present and compare in `is_cosmovisor_bump_needed' using string match